courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>pep8</li><li>I'd put the hyperparameters/constants (e.g., embedding_dim, untits, etc.) defined at the beginning of the cell in upper-case so that they are immediately recognizable from the rest of the code</li><li>Why not packaging the encoder,  decoder, and model code, into their own functions, say, create_encoder(...), create_decoder(...), create_model(...) (or classes possibly)? Is there a reason not to do that here?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>